test/data/yamlconfig.test

```diff
@@ -77,6 +77,7 @@
           },
           "Description": "libfetch before 2021-07-26, as used in apk-tools, xbps, and other products, mishandles numeric strings for the FTP and HTTP protocols. The FTP passive mode implementation allows an out-of-bounds read because strtol is used to parse the relevant numbers into address bytes. It does not check if the line ends prematurely. If it does, the for-loop condition checks for the '\\0' terminator one byte too late.",
           "Severity": "CRITICAL",
+          "Status": "fixed",
```

Comment:
@simar7 not sure why but my WFs are complaining for that line. Any advise why?

Reply:
what's WFs?